### PR TITLE
Do not expose supporting evidence if nil

### PR DESCRIPTION
**Pull request by timpeat**

### CODE REVIEW

**Review by hiboabd — APPROVED** [review]
Thank you!

File: app/api/datastore/entities/v1/crime_application.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ class CrimeApplication < BaseApplicationEntity
 
         expose :returned_at, expose_nil: false
         expose :return_details, expose_nil: false
-        expose :supporting_evidence
+        expose :supporting_evidence, expose_nil: false
```
Comment (zheileman) [inline app/api/datastore/entities/v1/crime_application.rb:11]: Would it make more sense to return empty array in the method we are using?

```ruby
        def supporting_evidence
          submitted_value('supporting_evidence')
        end
```

Comment (timpeat) [reply app/api/datastore/entities/v1/crime_application.rb:11]: good idea will sort

**Comment by hiboabd** [comment]
Closing to open another PR

**Pull request closed without merging**